examples/barotropicqg_betaforced.jl

```diff
@@ -55,7 +55,6 @@ kf, dkf = 14.0, 1.5     # forcing wavenumber and width of forcing ring in wavenu
 
 gr  = TwoDGrid(nx, Lx)
 
-x, y = gridpoints(gr)
 Kr = [ gr.kr[i] for i=1:gr.nkr, j=1:gr.nl]
```

Comment:
Why is this line here? I'm not sure where `Kr` is used, but the next line down seems unrelated.